consensus/src/runway/mod.rs

```diff
@@ -18,8 +18,10 @@ use futures::{
 use log::{debug, error, info, trace, warn};
 use std::{collections::HashSet, convert::TryFrom};
 
+mod backup;
 mod collection;
 
+pub use backup::{UnitLoader, UnitSaver, _run_loading_mechanism as run_backup_loading_mechanism};
```

Comment:
Why `pub`?

Reply:
So that UnitLoader and UnitSaver could be passed to runway as arguments. Do you think passing std::io::Read/Write to runway would be better? _run_loading_mechanism should not be here, I will remove it.

Reply:
Ah, yeah, all that makes sense.